### PR TITLE
Add frame types table to SITL doc

### PR DESCRIPTION
**Pull request by Hwurzburg**
baloon? plane-jet?

### CODE REVIEW

**Comment by khancyr** [comment]
hum, there is a lot more that those. Maybe we can generate it from the help ?

**Comment by Hwurzburg** [comment]
yeah....will do..I had used MP's list...will update,but I think I will restrict it to just vehicles, with a note on other frame types for specialized purposes see the help

File: dev/source/docs/using-sitl-for-ardupilot-testing.rst [hunk 1]
```diff
@@ -40,6 +40,51 @@ The frame type can also be changed with the ``-f`` parameter.
...
++                          + vtail quadplane-cl84   +
++                          + quadplane-tilthvec     +
++                          + quadplane-tilttri      +
++                          + quadplane-tilttrivec   +
++                          + quadplane-tri          +
++--------------------------+------------------------+
++                          + quad (default if -f    +
++                          + is not used)           +
++                          +------------------------+
++                          + coaxcopter djix dodeca-+
```
Comment (rmackay9) [inline dev/source/docs/using-sitl-for-ardupilot-testing.rst:72]: an extra space appears in "dodeca-hexa"

Comment (Hwurzburg) [reply dev/source/docs/using-sitl-for-ardupilot-testing.rst]: fixed

**Pull request merged**